facebook_importer.user.js

```diff
@@ -0,0 +1,74 @@
+// ==UserScript==
+// @name        MusicBrainz: Import events from Facebook
+// @version     2016-01-17
+// @author      Ohm Patel
+// @require     https://code.jquery.com/jquery-2.2.0.min.js
+// @grant       none
+// @include     *.facebook.*/events/*
+// ==/UserScript==
+
+var API_URL = "//graph.facebook.com/v2.5/";
+var API_KEY = "CAAO2IKDySlMBAOc6UU5jRHqiEFrENsZC09f4KJsMpDPOTV9gAjZC5tIdVqBeibxfk0i2XwuHL61ywnQdnI4KdCgwCfZCXwc0ZCryj48a1PRjVMuCbpoScvzeYlrTujLynPPKdgS66eLAvqGZCzl3Kj7ahRj97tYeR4VIGGVCQ9pRWDxZC9rPzIBEFTjvjBZCacZD";
```

Comment:
I think we had some discussion about this when @opatel99 wrote the script in January. @opatel99, can you remember what's going on here?